collect.py

```diff
@@ -203,6 +203,19 @@ def screenshot(driver, file_path):
     image.save(file_path)
 
 
+def get_code(driver, file_path):
+    if(not(os.path.isdir('source'))):
+        utils.mkdir('source')
+    source = driver.page_source
+    with open(file_path, "w", encoding="utf-8") as open_file:
+        open_file.append("source code : " + source + "\n")
+
+
+def get_screenshot_code(driver, location):
```

Comment:
**get_screenshot_code** is not a great name, it can be confusing.
Use **get_screenshot_and_domtree**.